packages/cli/src/cli/basemaps/action.tileset.info.ts

```diff
@@ -61,7 +61,11 @@ export class TileSetInfoAction extends TileSetBaseAction {
             console.log(this.renderHelpText());
             return;
         }
-        const tsData = await Aws.tileMetadata.TileSet.get(tileSet, projection, this.version.value! ?? TileSetTag.Head);
+        const tsData = await Aws.tileMetadata.TileSet.get(
+            tileSet,
+            projection,
+            this.version.value! ?? TileMetadataTag.Head,
```

Comment:
do you need a `!` if you are `??` it

Reply:
That was your code; not mine; I'll correct it

Reply:
Apparently we do need it?

Reply:
It is because of the overload of the get method and typescript not handling it properly